geode-assembly/src/test/java/org/apache/geode/rest/internal/web/controllers/RestAPIsWithSSLDUnitTest.java

```diff
@@ -585,21 +622,6 @@ public void testSSLWithTLSProtocol() throws Exception {
     validateConnection(restEndpoint, "TLS", props);
   }
 
-  @Test
-  public void testSSLWithTLSv11Protocol() throws Exception {
```

Comment:
This test should not be removed. Rather, mark as Ignore and provide Jira

Reply:
The problem is that all cipher suites for TLSv11 have known vulnerabilities and have been disabled by Jetty.  If we bump the Jetty version we are essentially removing support for TLS v1.1.

Reply:
Regardless, it's still possible to do it (even if it's bad). Some people will need it for older clients that can only do TLSv1.1.